Shokofin/Providers/MovieProvider.cs

```diff
@@ -0,0 +1,134 @@
+
+                _logger.LogInformation($"Shoko Scanner... Getting movie ID ({filename})");
+
+                var apiResponse = await ShokoAPI.GetFilePathEndsWith(filename);
+                var file = apiResponse?.FirstOrDefault();
+                var fileId = file?.ID.ToString();
+                var seriesIds = file?.SeriesIDs.FirstOrDefault();
+                var seriesId = seriesIds?.SeriesID.ID.ToString();
+                var episodeIds = seriesIds?.EpisodeIDs?.FirstOrDefault();
+                var episodeId = episodeIds?.ID.ToString();
+
+                if (string.IsNullOrEmpty(episodeId) || string.IsNullOrEmpty(seriesId))
+                {
+                    _logger.LogInformation($"Shoko Scanner... File not found! ({filename})");
+                    return result;
+                }
+
+                _logger.LogInformation($"Shoko Scanner... Getting movie metadata ({filename} - {episodeId})");
+
+                var seriesAniDB = await ShokoAPI.GetSeriesAniDb(seriesId);
+                var series = await ShokoAPI.GetSeries(seriesId);
+                var episodeAniDB = await ShokoAPI.GetEpisodeAniDb(episodeId);
+                var episode = await ShokoAPI.GetEpisode(episodeId);
+                bool isMultiEntry = series.Sizes.Total.Episodes > 1;
+                int aniDBId = (isMultiEntry ? episodeIds?.AniDB : seriesIds?.SeriesID.AniDB) ?? 0;
+                int tvdbId = (isMultiEntry ? episodeIds?.TvDB?.FirstOrDefault() : seriesIds?.SeriesID.TvDB?.FirstOrDefault()) ?? 0;
+
+                if (seriesAniDB?.SeriesType != "0")
+                {
+                    _logger.LogInformation($"Shoko Scanner... File found, but not a movie! Skipping.");
+                    return result;
+                }
+
+                var tags = await ShokoAPI.GetSeriesTags(seriesId, Helper.GetFlagFilter());
+                var ( displayTitle, alternateTitle ) = Helper.GetMovieTitles(seriesAniDB.Titles, episodeAniDB.Titles, series.Name, episode.Name, Plugin.Instance.Configuration.TitleMainType, Plugin.Instance.Configuration.TitleAlternateType, info.MetadataLanguage);
+                float comRat = isMultiEntry ? (float)((episodeAniDB.Rating.Value * 10) / episodeAniDB.Rating.MaxValue) : (float)((seriesAniDB.Rating.Value * 10) / seriesAniDB.Rating.MaxValue);
+                ExtraType? extraType = Helper.GetExtraType(episodeAniDB);
+
+                result.Item = new Movie
+                {
+                    IndexNumber = Helper.GetIndexNumber(series, episodeAniDB),
+                    Name = displayTitle,
+                    OriginalTitle = alternateTitle,
+                    PremiereDate = isMultiEntry ? episodeAniDB.AirDate : seriesAniDB.AirDate,
+                    // Use the file description if collection contains more than one movie, otherwise use the collection description.
+                    Overview = Helper.SummarySanitizer((isMultiEntry ? episodeAniDB.Description : seriesAniDB.Description) ?? ""),
```

Comment:
Make it fallback to the series description if the episode description is null before falling back to empty.
So, if it's a multi-entry. If ep desc is null, falls back to series desc. If series desc is also null, then falls back to the empty string.